docs/releases/4.1.txt

```diff
@@ -188,6 +188,9 @@ Forms
   labels in ``<legend>`` tags via the new ``tag`` argument of
   :meth:`~django.forms.BoundField.label_tag`.
 
+* To improve accessibility :attr:`~django.forms.Field.help_text` is now
```

Comment:
I would suggest expanding upon this to clarify it’s not just an arbitrary enhancement but fixes a real-world accessibility issue with help text not being programmatically identifiable.